bpf/bpf_host.c

```diff
@@ -1242,6 +1242,9 @@ int to_netdev(struct __ctx_buff *ctx __maybe_unused)
 		break;
 	}
 
+        send_trace_notify(ctx, TRACE_TO_NETWORK, srcID, 0, 0,
+              0, ret, 0);
```

Comment:
The indentation is incorrect here.

Reply:
This trace won't match the reverse path since on `from-netdev` we trace packets with `TRACE_FROM_STACK` (see `do_netdev()`). Maybe we should fix that later trace to be `TRACE_FROM_NETWORK` as well?
/cc @cilium/hubble

Reply:
I'm not aware of any code Hubble that would break if we changed the trace point for `from-netdev`.

Reply:
- Fixed indentation

- Changed TRACE_FROM_STACK to TRACE_FROM_NETWORK in do_netdev()

PTAL :)

(Sorry I've been diverted a lot, will be more responsive)

Reply:
No problem :+1: